common-features/extensions-ai-integration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Integration with Microsoft.Extensions.AI
+page_title: Integration with Microsoft.Extensions.AI
+description: How to integrate the UI for Blazor components with Microsoft.Extensions.AI
+slug: common-features-extensions-ai-integration
+tags: telerik,blazor,aiprompt,ai,extensions,integration
+published: True
+position: 4
+---
+
+# Integration with Microsoft.Extensions.AI
+
+Some Telerik UI for Blazor components incorporate the [Microsoft.Extensions.AI package](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.ai?view=net-9.0-pp) to simplify your AI integration, provide flexibility and let you easily use and test various AI providers.
```

Comment:
Better start with the specific components. Such a general statement may raise eyebrows in readers who don't want any AI stuff. We already had tickets about this.